### DISCUSSION TITLE
Schneider/Wiser CCTFR6600 support

### DISCUSSION THREAD

**Discussion by quiet2754** [Koenkk/zigbee2mqtt#23976] [Devices]
Hello,

The device Schneider CCTFR6600 is not currently supported by Zigbee2mqtt but CCTFR6700 (very similay / water vs electrical heating) is.
Does it mean that CCTFR6600 will not be added and manageable at all ?
If yes, what is the way to add a new device to the supported list ?

I will have 1x CCTFR6600 linked via Zigbee to 6x CCTFR6400.
Is it possible to have those devices linked between each other via Zigbee AND linked to a previous zigbee2mqtt network ?
I am afraid that they will have a dedicated Zigbee-Schneider network without possibility to add them to my current Zigbee network.

Thank you for your answers :)

**Comment by Koenkk** [comment]
Does it work when stopping z2m, in the data/database.db change `CCTFR6600` to `CCTFR6700` and start z2m?

**Reply by quiet2754** [reply]
I don't know. I am requesting before buying it to understand if I can choose this product :)
